analytics/lib/counts.py

```diff
@@ -293,7 +296,7 @@ def do_aggregate_to_summary_table(
 
 # called from zerver.actions; should not throw any errors
 def do_increment_logging_stat(
-    zerver_object: Union[Realm, UserProfile, Stream],
+    zulip_object: Union[Realm, UserProfile, Stream, "RemoteZulipServer"],
```

Comment:
Let's do a prep commit just renaming this parameter to something like `model_object_for_bucket`? I feel like that might better hint what this is for.